tests/mbed/host/host.c

```diff
@@ -37,20 +37,14 @@ char* find_data_file(char* str, size_t size)
 void datafileloc(char* data_file_name, char* path)
 {
     char* tail = "3rdparty/mbedtls/mbedtls/tests/suites/";
-    char* seperator;
+    char* seperator = getenv("PROJECT_SOURCE_DIR");
```

Comment:
Can you please fix the spelling for seperator to separator (if possible)? This is in existing code.

Reply:
Ofc. Sorry I missed it before. I updated the code.